sentry/src/main/java/io/sentry/instrumentation/file/FileIOSpanManager.java

```diff
@@ -112,6 +112,22 @@ private void finishSpan() {
     }
   }
 
+  private @NotNull String getDescription(final @NotNull File file) {
+    final String byteCountToString = StringUtils.byteCountToString(byteCount);
+    // if we send PII, we can send the file name directly
+    if (options.isSendDefaultPii()) {
+      return file.getName() + " (" + byteCountToString + ")";
+    }
+    final int lastDotIndex = file.getName().lastIndexOf('.');
```

Comment:
Can we use `StringUtils.getStringAfterDot` here instead?

Reply:
Ah I just noticed, it returns the full string, if there's no dot, nevermind then.

Reply:
i want to use the extension if available, otherwise "***"
with `StringUtils.getStringAfterDot` i'd have to check if the returned string is different from the file name in the `if` condition, so no real gain from it imho